src/app/producer-shelf/models/key-binding.ts

```diff
@@ -0,0 +1,7 @@
+export interface KeyBinding {
+  readonly key: string
+  readonly modifiers: string[]
+  readonly label: string
+  readonly action: () => void
+  readonly onKeyPress: boolean
```

Comment:
`onKeyPress` is a common term for the function that should be executed on the press of the key.
Here it is a boolean which is a bit confusing. What does it mean when it is true, and what does it mean when it is false? That is not clear to me.

Personally, I would prefer `onKeyPress` to be the name for the attribute containing the function to be executed instead of `action`.
`action` is a bit vague and since we have a concept of `Action` in the codebase it is easy to confuse this with being an attribute that holds an actual Action, which it does not.

Reply:
Yes, i can delete it here and scrub out most of the things not currently used. But it will happen in the PR right after. 
Should i delete all these thing and let @KvelaGorrrrnio introduce it all instead?

Reply:
yes

Reply:
Thy will be done. It has been removed!